libs/ColorKit/source/ColorKit.cpp

```diff
@@ -1,15 +1,50 @@
 // Leka - LekaOS
 // Copyright 2022 APF France handicap
 // SPDX-License-Identifier: Apache-2.0
+#pragma once
 
 #include "ColorKit.h"
 #include <cmath>
 
+#include "ColorPosition.h"
+#include "Utils.h"
+
 namespace leka {
 
 auto ColorKit::colorGradient(const RGB &color1, const RGB &color2, const float &position) -> RGB
 {
 	return oklab2rgb(lerp(ColorKit::internal::rgb2oklab(color1), ColorKit::internal::rgb2oklab(color2), position));
 }
 
+auto ColorKit::create255RGBGradient(std::vector<ColorPosition> input_colors, const RGB &color_start,
+									const RGB &color_end) -> std::array<RGB, 255>
+{
+	std::array<RGB, 255> return_colors;
+	auto size		 = static_cast<uint8_t>(input_colors.size());
+	auto first_input = input_colors[0];
+
+	auto last_input = input_colors[size - 1];
+
+	for (uint8_t index(0); index < first_input.position; index++) {
```

Comment:
```suggestion
	for (auto i = 0; i < first_input.position; ++i) {
```

c'est toujours `++i` que tu veux, jamais `i++`-- tu connais la différence entre les deux?